src/admin/permission-groups/views/PermissionGroupEdit.tsx

```diff
@@ -0,0 +1,426 @@
+				addedPermissions.map(p => p.id),
+				permissionGroupId
+			);
+			await PermissionGroupService.removePermissionsToGroup(
+				removedPermissions.map(p => p.id),
+				permissionGroupId
+			);
+
+			ToastService.success(t('De permissie groep is opgeslagen'), false);
+			setIsSaving(false);
+		} catch (err) {
+			console.error(
+				new CustomError('Failed to save permission group', err, {
+					permissionGroup,
+					initialPermissionGroup,
+				})
+			);
+			ToastService.danger(t('Het opslaan van de permissiegroep is mislukt'), false);
+		}
+	};
+
+	const getAllPermissions = () => {
+		const permissionIdsInGroup: number[] = (get(
+			permissionGroup,
+			'permissions',
+			[]
+		) as Permission[]).map(p => p.id);
+		return allPermissions
+			.filter(p => {
+				// Don't show permissions that are already part of the permission group permissions
+				return !permissionIdsInGroup.includes(p.id);
+			})
+			.map(p => ({
+				label: p.description || p.label,
+				value: String(p.id),
+			}));
+	};
+
+	const renderTableCell = (rowData: Permission, columnId: PermissionsTableCols) => {
+		switch (columnId) {
+			case 'label':
+			case 'description':
+				return rowData[columnId];
+
+			case 'actions':
+				return (
+					<ButtonToolbar>
+						<Button
+							icon="delete"
+							onClick={() => deletePermission(rowData.id)}
+							size="small"
+							ariaLabel={t('Verwijder')}
+							title={t('Verwijder')}
+							type="tertiary"
+						/>
+					</ButtonToolbar>
+				);
+
+			default:
+				return rowData[columnId];
+		}
+	};
+
+	const renderEditPage = () => {
+		if (!permissionGroup) {
+			return;
+		}
+		return (
+			<>
+				<Container size="medium">
+					<Spacer margin="bottom-extra-large">
+						<Box backgroundColor="gray">
+							<Form>
+								<FormGroup label={t('Label')} error={formErrors.label}>
+									<TextInput
+										value={permissionGroup.label || ''}
+										onChange={newLabel =>
+											setPermissionGroup({
+												...permissionGroup,
+												label: newLabel,
+											})
+										}
+									/>
+								</FormGroup>
+								<FormGroup label={t('Beschrijving')} error={formErrors.description}>
+									<TextInput
+										value={permissionGroup.description || ''}
+										onChange={newDescription =>
+											setPermissionGroup({
+												...permissionGroup,
+												description: newDescription,
+											})
+										}
+									/>
+								</FormGroup>
+							</Form>
+						</Box>
+					</Spacer>
+				</Container>
+
+				<Panel>
+					<PanelHeader>
+						<BlockHeading type="h3">Permissies in deze groep:</BlockHeading>
+					</PanelHeader>
+					<PanelBody>
+						<Spacer margin="bottom-large">
+							<Form type="inline">
+								<FormGroup label={t('Voeg een permissie toe aan deze groep: ')}>
+									<Select
+										options={getAllPermissions()}
+										placeholder={t('Kies een permissie')}
+										value={selectedPermissionId}
```

Comment:
Crasht hier: `Type 'string | null' is not assignable to type 'string | undefined'.`